samples/cicd/skaffold/README.md

```diff
@@ -0,0 +1,45 @@
+# Skaffold
+
+This is intended for demonstration only, and is not tuned for performance or security.
+
+skaffold is a tool that enables fast development iteration and controls deployment to local or remote clusters
+
+If running `skaffold run` for deployment, manifests are pulled from remote charts, if running `skaffold dev` for development and hot reload, manifests are pulled from current branch.
+
+## Quick Start
+
+skaffold is built around modules and profiles
+
+1) istio-base + istio
```

Comment:
I would have a single skafold at base, for istiod only. 

We could add a second one for gateway, but please don't mix them. 

Eventually gateways will be auto-provisioned,

Reply:
Never mind, they are separate configs inside same file.

Still would be nice to split the files - but ok to merge it as one file too.